notebooks/06_validate.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>Is it better to print <code>Annotation(id='2202383')</code> to show why it has <code>NegativeLength</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>